credentialsdemo/source/no-sdk/src/org/kaaproject/kaa/examples/credentials/utils/CredentialsConstants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kaaproject.kaa.examples.credentials.utils;
+
+/**
+ * @author Maksym Liashenko
+ */
+public class CredentialsConstants {
```

Comment:
I think this class is redundant in a scope of the sample application, move all constants in the `KaaAdminManager.java` and remove this class.